src/shogun/preprocessor/Preprocessor.h

```diff
@@ -40,7 +40,7 @@ enum EPreprocessorType
 	P_DECOMPRESSBYTESTRING=90,
 	P_DECOMPRESSWORDSTRING=100,
 	P_DECOMPRESSULONGSTRING=110,
-	P_RANDOMFOURIERGAUSS=120,
+	P_RFF=120,
```

Comment:
this is an error that pops up at *compile time* 
Always make sure that your code compiles before pushing pls. Better yet, run the (relevant) tests. Otherwise you just waste energy with our CI doing a lot of pointless computation